tools/ratbagctl.in

```diff
@@ -719,7 +719,7 @@ parser_def = [
                     func: func_led_get,
                 },
                 {
-                    of_type: set,
+                    of_type: switch,
```

Comment:
NACK on this one.

A `set` allows to have more than one command one after the other (like `mode cycle brightness 255`) while a `switch` has only one possible value (so `mode cycle brightness 255` fails).

This can be checked by the fact that the test suite now fails.

Which device/command are you trying to fix?